KarhooUISDK/Screens/CheckoutScreen/Components/AddPaymentView/KarhooAddPaymentPresenter.swift

```diff
@@ -13,7 +13,7 @@ final class KarhooAddPaymentPresenter: AddPaymentPresenter {
 
     private let analyticsService: AnalyticsService
     private let userService: UserService
-    private let view: AddPaymentView
+    private unowned let view: AddPaymentView
```

Comment:
Need to be careful with this when testing

Reply:
It is owned by the view already